pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -69,8 +69,8 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 		projectDisplayName = projectRequest.DisplayName
 	}
 
-	if len(projectRequest.Annotations["description"]) > 0 {
-		projectDescription = projectRequest.Annotations["description"]
+	if len(projectRequest.Annotations[projectapi.ProjectDescription]) > 0 {
```

Comment:
@liggitt in createProject.js - currently why is `description` currently not an inline string? Also - should we be passing in the `displayName` as it's own parameter or moving it into the annotation list?